src/pages/ScreeningsPage.vue

```diff
@@ -1,11 +1,156 @@
 <template>
-  <div></div>
+  <div>
+    <h1>Screenings</h1>
+    <h2>{{ selectedDay }}</h2>
+    <div>
+      <button
+        class="day-tabs"
+        v-for="day in daysList"
+        :key="day.id"
+        :data-date="day.date.toJSON()"
+        @click="setSelectedDay"
+      >
+        {{ day.id === 1 ? day.name : niceDay(day.name) }}
+      </button>
+      <button class="day-tabs__calendar">
+        <img src="@/assets/img/calendaricon.svg" alt="calendar-icon" />
+      </button>
+      <hr />
+      <div v-for="movieId in moviesOnTheDay" :key="`movie_${movieId}`">
+        <p>
+          {{
+            $store.getters.movieList.filter((movie) => movie.id == movieId)[0]
+              .title
+          }}
+        </p>
+        <p
+          v-for="seance in seancesList
+            .filter(
+              (seance) => seance.datetime.substring(0, 10) === selectedDay
+            )
+            .filter((seance) => seance.movie == movieId)"
+          :key="`seance_${seance.id}`"
+        >
+          {{ seance.datetime.substring(11, 16) }}
+        </p>
+      </div>
+    </div>
+  </div>
 </template>
 
 <script>
+import apiSeancesService from "@/services/api/apiSeancesService";
+
 export default {
   name: "ScreeningsPage",
+  data() {
+    return {
+      daysList: [],
+      selectedDay: "",
+      seancesList: [],
+      seanceOnTheDay: [],
+      moviesOnTheDay: [],
+    };
+  },
+  computed: {
+    todayDate() {
+      return new Date();
+    },
+    todayDaySelection() {
+      let date = new Date();
+      return date.toJSON().substring(0, 10);
+    },
+  },
+  methods: {
+    niceDay(dayValue) {
+      switch (dayValue) {
+        case 1:
+          return "Mon";
+        case 2:
+          return "Tue";
+        case 3:
+          return "Wed";
+        case 4:
+          return "Thu";
+        case 5:
+          return "Fri";
+        case 6:
+          return "Sat";
+        case 0:
+          return "Sun";
+      }
+    },
+    setSelectedDay(event) {
+      this.selectedDay = event.currentTarget
+        .getAttribute("data-date")
+        .substring(0, 10);
+      console.log(this.selectedDay);
+      this.setMoviesOnTheDay();
+    },
+    setMoviesOnTheDay() {
+      const seancesOnTheDay = this.seancesList.filter(
+        (seance) => seance.datetime.substring(0, 10) === this.selectedDay
+      );
+      const uniqueMovies = [
+        ...new Set(seancesOnTheDay.map((seance) => seance.movie)),
+      ];
+      this.moviesOnTheDay = uniqueMovies;
+    },
```

Comment:
same here - it can be a computed value (e.g. `moviesOnTheDay`) that depends on `this.selectedDay` and `this.seancesList`. Then you won't need to manually call `setMoviesOnTheDay()` and `moviesOnTheDay` will be automatically computed for you every time above mentioned dependencies are changed.